schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+/* Database schema to keep the structure of the entire database. */
+CREATE TABLE animals(
+   id serial primary key,
+   name CHAR(20),
+   date_of_birth DATE,
+   escape_attempt INT,
+   neutered BOOL,
+   weight_kg DECIMAL   
+); 
+
+CREATE TABLE owners(
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(50),
+  age INT  
+);
+
+CREATE TABLE species(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50)  
+);
+
+CREATE TABLE vets(
+   id SERIAL PRIMARY KEY,
+   name VARCHAR(50),
+   age INT,
+   date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+	vet_id INT NOT NULL, 
+	species_id INT NOT NULL, 	
+	CONSTRAINT fk_vet FOREIGN KEY (vet_id) 
+	REFERENCES vets(id),
+	CONSTRAINT fk_species FOREIGN KEY(species_id) 
+	REFERENCES species(id));
+	
+CREATE TABLE visits (
+	vet_id INT NOT NULL, 
+	animals_id INT NOT NULL, 
+	date_of_visit DATE,	
+	CONSTRAINT fk_vet FOREIGN KEY (vet_id) 
+	REFERENCES vets(id),
+	CONSTRAINT fk_animals FOREIGN KEY(animals_id) 
+	REFERENCES animals(id));
+
+
```

Comment:
- [ ] Please create tables based on the diagram provided in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md). None of the above tables exists in the diagram.